docs/roadmap.md

```diff
@@ -7,78 +7,114 @@ The following roadmap gives an overview of what we are currently working on and
 
 ## Current focus
 
-**Search & Resource page redesign**<br/>
+#### **Search & Resource page redesign**<br/>
 *What*: Redesign the search experience and the resource page, to make them scalable in the number of resources types and the number of metadata<br/>
 *Status*: Designs are ready, engineering work has started<br/>
 *Links*: [Designs](https://drive.google.com/drive/folders/12oBrcXUsDtOsuU_QvO93LTvs4Dehx6az?usp=sharing)
+<br/>
+<br/>
 
-**Email notifications system**<br/>
+#### Email notifications system
 *What*: We are creating an email notification system to reach Amundsen’s users. The primary goal is to use this system to help solve the lack of ownership for data assets at Lyft. The secondary goal is to engage with users for general purposes.<br/>
 *Status*: Designs are ready, engineering work has started
+<br/>
+<br/>
 
 ## Next steps
 
-**Index Dashboards**
+#### Index Dashboards
 *What*: We want to help with the discovery of existing analysis work, dashboards. This is going to help avoid reinventing the wheel, create value for less technical users and help give context on how tables are used.<br/>
 *Status*: Product + technical specs are ready, designs are ready, implementation has not started<br/>
 *Links*: [Designs](https://drive.google.com/drive/folders/12oBrcXUsDtOsuU_QvO93LTvs4Dehx6az?usp=sharing) | [Product Specifications](https://docs.google.com/document/d/16cSKgM2sCYvhKq54yfwaHKwslJEGtdS2g5dcPV4p5qo/edit?usp=sharing) | [Technical RFC](https://docs.google.com/document/d/1PHk8OjcIULJ7hG0ckeMrRfTk3vXqnq5asEykgQUw-Ow/edit?usp=sharing)
+<br/>
```

Comment:
and I don't think we need </br> for a line break in markdown file? we only need a space which should do the same.

Reply:
In general you’re absolutely right. I did it here to add some extra white space after the short paragraphs to visually have the headings closer to the paragraph they head.

If you still prefer it without BRs I will change it.

Reply:
Duh - had to re-read what you wrote ... and you're right. Though it has to be trailing double-spaces